### PR TITLE
Add test data for holdout assignments

### PR DESCRIPTION
**Pull request by sameerank**

### CODE REVIEW

File: assignment-v3-beta/test-case-7.json [hunk 1]
```diff
@@ -0,0 +1,2009 @@
+{
+    "experiment": "experiment_with_holdout",
+    "valueType": "string",
+    "subjects": [
+      "6255e1a70560594e38edcb55",
```
Comment (sameerank) [inline assignment-v3-beta/test-case-7.json:5]: Personally, I find this list of a 1k subjects to be opaque when it comes to debugging, but these are the same subjects used in `randomization_algo` when the `expectedAssignments` updated to match holdouts

Comment (leoromanovsky) [reply assignment-v3-beta/test-case-7.json]: Ya this is a big file - can it be accomplished with fewer subjects?

**Review by leoromanovsky — APPROVED** [review]

**Comment by aarsilv** [comment]
I got a bunch of comments and questions in https://github.com/Eppo-exp/js-client-sdk-common/pull/27; once I understand those, then I can intelligently comment on this PR 😛

File: rac-experiments-v3-hashed-keys.json [hunk 2]
```diff
@@ -341,6 +362,81 @@
...
+          "allocationKey": "allocation-experiment-1",
+          "conditions": []
+        }
+      ],
+      "allocations": {
+        "allocation-experiment-1": {
+          "percentExposure": 0.4533,
+          "statusQuoVariationKey": "variation-7",
+          "shippedVariationKey": "variation-8",
+          "holdouts": [
```
Comment (aarsilv) [inline rac-experiments-v3-hashed-keys.json:382]: 🔥

File: rac-experiments-v3-hashed-keys.json [hunk 3]
```diff
@@ -341,6 +362,81 @@
...
+            {
+              "holdoutKey": "holdout-3",
+              "statusQuoShardRange": {
+                "start": 200,
+                "end": 300
+              },
+              "shippedShardRange": {
+                "start": 300,
+                "end": 400
+              }
```
Comment (aarsilv) [inline rac-experiments-v3-hashed-keys.json:403]: Consider making these numbers more random (complete with gaps between them) to reflect our planned strategy

File: rac-experiments-v3-obfuscated.json [hunk 4]
```diff
@@ -345,6 +369,123 @@
           ]
         }
       }
+    },
+    "61c379905d979d2d2e42208ccd900680": {
```
Comment (aarsilv) [inline rac-experiments-v3-obfuscated.json:373]: thanks for adding the bandit on in here!

File: rac-experiments-v3.json [hunk 5]
```diff
@@ -387,6 +411,81 @@
           ]
         }
       }
+    },
+    "experiment_with_holdout": {
```
Comment (aarsilv) [inline rac-experiments-v3.json:415]: 🙌

**Review by aarsilv — APPROVED** [review]
Nice work!

File: rac-experiments-v3-hashed-keys.json (hunk 3 above)
Comment (sameerank) [reply rac-experiments-v3-hashed-keys.json:403]: Ah yes, good catch!

**Pull request merged**